### DISCUSSION TITLE
Speedcoaches

### DISCUSSION THREAD

**Discussion by NickStijger** [aivd-asopos/.github#1] [Ideas]
Zelf speedcoaches maken? :O

**Comment by NickStijger** [comment]
Mogelijkheid 1:
embedded, dus los apparaat

**Comment by NickStijger** [comment]
Mogelijkheid 2:
onderdeel van de av app

**Reply by joostvhts** [reply]
check deze https://github.com/mdjong1/Talos-Rowing-2 een proteeër (waarmee ik toen ff contact had vm in een issue) die een bestaand ding enigszins uptodate heeft gemaakt waar we misschien dingen uit kunnen gebruiken

**Reply by NickStijger** [reply]
Interessant, mogelijk code om als basis, maar tis wel java helaas
